### PR TITLE
Help info 

### PR DESCRIPTION
**Pull request by MohamedOmarii**
related to #106

### CODE REVIEW

File: text.js [hunk 1]
```diff
@@ -15,6 +16,7 @@ module.exports = {
     back: 'العوده صفحه الى الوراء ',
     homePage: 'العوده الى الصفحه الرئيسيه ',
     homeIcon: ' ايقونه الصفحه الرئيسيه ',
-    helpIcon: ' ايقونه للمساعده'
+    helpIcon: ' ايقونه للمساعده',
+    helpInfo: 'يرصد إيسيناز في تهدف إلى السماح للناس يعرفون من عدد قليل من الأماكن التي يمكن الوصول إليها في الناصرة وتشجيع المزيد من الشركات لتصبح في متناول!    في التطبيق لدينا يمكنك أن تجد جميع الأماكن التي يمكن الوصول إليها والشركات في نازارث في أي وقت وعدد قليل من النقرات يعتمد على ما تحتاجه أو تبحث عن'
```
Comment (Suuuuha) [inline text.js]: lol .. google translate? :-1: 
maybe just write one or two sentences about the app in Arabic from what you know about it, you don't need to have the exact translate from the english app info

Comment (MohamedOmarii) [inline text.js]: maybe you can to translate this

**Comment by esraajb** [comment]
reference an issue

**Comment by MohamedOmarii** [comment]
related to #106

**Comment by esraajb** [comment]
@MohamedOmarii the issue should be in the body of the first comment

**Comment by esraajb** [comment]
@MohamedOmarii fix merge conflicts

**Comment by esraajb** [comment]
@MohamedOmarii the naming makes more sense now. well done 👍

**Pull request merged**